content/en/account_management/billing/usage_attribution.md

```diff
@@ -85,6 +91,23 @@ This section provides daily reports at an hourly granularity to dig into time fr
 
 Daily data can also be pulled using the tool's public API. For more information, see the [API endpoint documentation][2].
 
+[2]: https://docs.datadoghq.com/api/v1/usage-metering/#get-hourly-usage-attribution
+
+
+{{< /site-region >}}
+{{< site-region region="us3,us5,gov" >}}
+
+<div class="alert alert-info">
+Daily usage attribution is only available for organizations in a US1 or EU1 datacenter.
```

Comment:
```suggestion
Daily usage attribution is only available for organizations in a US1 or EU1 datacenter. On US3, US5, and FedGov, you can instead download a report.
```

Reply:
I added this because otherwise it seems vague as to whether these customers, who might not know exactly what "Daily usage attribution" looks like, can do the next steps (they can).

Reply:
@kayayarai what specifically you referring to for "download a report"? US3, US5 and FedGov customers cannot download the daily usage attribution reports because the feature is deprecated for them. However, they are still able to download the Monthly usage attribution data via CSV or access hourly usage attribution data via API